src/shared/api.ts

```diff
@@ -575,6 +575,30 @@ export const vertexModels = {
 			},
 		],
 	},
+	"gemini-2.5-pro-preview-06-05": {
```

Comment:
<img width="877" alt="image" src="https://github.com/user-attachments/assets/8f165968-c8e4-4bbe-b4c5-2e59077a859b" />
vertex does support this already